frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/utils.ts

```diff
@@ -18,6 +18,9 @@ import { CREATE_PIPELINE_RESOURCE, initialResourceFormValues } from './const';
 import { CommonPipelineModalFormikValues, PipelineModalFormResource } from './types';
 import { TektonResourceLabel, VolumeTypes } from '../../const';
 
+// Annotation for referencing pipeline name in case of PipelineRun with no reference to a Pipeline (embedded pipeline)
+export const preferredNameAnnotation = 'io.openshift.pipeline/preferredName';
```

Comment:
We have an  **started-by** annotations already, it is like this  `pipeline.openshift.io/started-by`. We should be consistent here
```suggestion
export const preferredNameAnnotation = 'pipeline.openshift.io/preferredName';
```
and also move this to `src/components/pipelines/const.ts`

Reply:
done